projects/kit/components/input-range/input-range.template.html

```diff
@@ -86,6 +86,7 @@
         [ngModel]="value"
         (ngModelChange)="onRangeValue($event)"
         (tuiPressedChange)="focusToTextInput()"
+        (touchend)="blurRange()"
```

Comment:
unfortunately cypress doesn't have mobile emulation (we can only set mobile viewport).
I have tried to "fake" mobile now via
```ts
cy.get('@leftSlider')
  .trigger('touchstart', 0, 0, {force: true})
  .trigger('touchmove', 300, 0, {force: true})
  .trigger('touchend', 300, 0, {force: true});
```
But it doesn't properly works(

As for unit tests, they are useless here because they will test native-browser's things (

Reply:
Don't worry!